src/routes/stops/[stopID]/schedule/+page.svelte

```diff
@@ -143,22 +152,21 @@
 				</h2>
 
 				<div class="mb-4 flex gap-4">
-					<div class="min-w-32">
+					<div class="z-20 w-[30%] min-w-32">
```

Comment:
this causes a regression on mobile devices.

<img width="429" alt="image" src="https://github.com/user-attachments/assets/2beb2f81-24b6-471e-a326-25f9c2f9a7a2" />

Reply:
Okay. Thanks for solving this :)